exercise/templates/exercise/staff/results.html

```diff
@@ -160,14 +160,21 @@ <h4>{% trans "Summaries" %}</h4>
 
 		<!--Dropdown button for table exports. The buttons are added inside results_staff.js-->
 		<div id="table-export-dropdown" class="dropdown d-inline-block">
-			<button class="btn btn-primary dropdown-toggle" type="button" data-toggle="dropdown">{% trans "Export Data" %}
+			<button disabled class="btn btn-primary dropdown-toggle" type="button" data-toggle="dropdown">{% trans "Export Data" %}
 				<span class="caret"></span>
 			</button>
 			<ul id="export-button-menu" class="dropdown-menu">
 
 			</ul>
 		</div>
 
+		<!--Alerts-->
+		<div id="alerts" style="margin: 30px">
+			<div id="ajax-failed-alert" class="alert alert-danger collapse" role="alert">
```

Comment:
`hidden` would probable be a better class or did `collapse` look better?

Reply:
Hidden doesn't work, apparently.